lib/WUI/link_content/basic_gets.cpp

```diff
@@ -249,4 +253,29 @@ JsonResult get_job(size_t resume_point, JsonOutput &output) {
     // clang-format on
 }
 
+bool usb_available() {
+    bool available = false;
+    // ideally we would use something more lightweight, like stat()
+    // but fatfs doesn't support calling it on root and from it's
+    // perspective /usb is root
+    if (DIR *dir = opendir("/usb"); dir != nullptr) {
+        available = true;
+        closedir(dir);
+    }
+    return available;
+}
+
+JsonResult get_storage(size_t resume_point, JsonOutput &output) {
```

Comment:
I'm sorry to still bother with minor details. But the function is used only here, locally. Can you put it into an anonymous namespace, so it doesn't become linkable and can't collide with some other `get_storage` function somewhere else?

Reply:
I don't get this, `get_storage` is used in `prusa_link_api.cpp` don't you mean the function `usb_available` above?

Reply:
Yes, of course. I've attached it to the wrong function.